### PR TITLE
#2065 Increase match report column size in database

### PR DESCRIPTION
**Pull request by wsbrenk**
1. changes proposed in this pull request:

   - fixes issue #2065
   - if not fixing an existing issue, comments explaining the purpose of the PR should be provided)

2. `src/main/resources/release_notes.md` ...

 - [x] has been updated
 - [ ] does not require update

3. [Optional] suggested person to review this PR @tychobrailleur

### CODE REVIEW

File: src/main/java/core/db/AbstractTable.java [hunk 1]
```diff
@@ -44,6 +44,17 @@ public AbstractTable(String tableName, ConnectionManager connectionManager) {
 		initColumns();
 	}
 
+	/**
+	 * Truncate string to maximum length
+	 * @param s String is truncated if it's length exceeds given limit
```
Comment (tychobrailleur) [inline src/main/java/core/db/AbstractTable.java:49]: “its”

File: src/main/java/core/db/AbstractTable.java [hunk 2]
```diff
@@ -44,6 +44,17 @@ public AbstractTable(String tableName, ConnectionManager connectionManager) {
...
+	 * @param maxLength Length limit
+	 * @return Truncated string
+	 */
+	static String TruncateString(String s, int maxLength) {
```
Comment (tychobrailleur) [inline src/main/java/core/db/AbstractTable.java:53]: Would you mind using lowercase for the method name?  `truncateString`

**Review by tychobrailleur — APPROVED** [review]

File: src/main/java/core/db/AbstractTable.java (hunk 1 above)
Comment (wsbrenk) [reply src/main/java/core/db/AbstractTable.java:49]: urgh - fixed

File: src/main/java/core/db/AbstractTable.java (hunk 2 above)
Comment (wsbrenk) [reply src/main/java/core/db/AbstractTable.java]: fixed - too much c++-programming in the last time....

**Review by tychobrailleur — APPROVED** [review]

File: src/main/java/core/db/AbstractTable.java (hunk 2 above)
Comment (tychobrailleur) [reply src/main/java/core/db/AbstractTable.java]: I suspected as much, there are telltale signs! ;-)
Thanks for taking the time to address this.

**Pull request merged**